capabilities.go

```diff
@@ -137,7 +149,61 @@ func (c *Client) GetCapabilities(target string, port int) (response *Capabilitie
 	// Invalid version detected
 	if len(response.BsvAlias) == 0 {
 		err = fmt.Errorf("missing %s version", DefaultServiceName)
+		return
+	}
+
+	// Parse PIKE capability
+	if err = parsePikeCapability(response); err != nil {
+		return
 	}
 
 	return
 }
+
+// ExtractPikeOutputsURL extracts the outputs URL from the PIKE capability
+func (c *CapabilitiesPayload) ExtractPikeOutputsURL() string {
+	if c.Pike != nil {
+		return c.Pike.Outputs
+	}
+	return ""
+}
+
+// ExtractPikeInviteURL extracts the invite URL from the PIKE capability
+func (c *CapabilitiesPayload) ExtractPikeInviteURL() string {
+	if c.Pike != nil {
+		return c.Pike.Invite
+	}
+	return ""
+}
+
+// parsePikeCapability parses the PIKE capability from the capabilities response
+func parsePikeCapability(response *CapabilitiesResponse) error {
+	if pike, ok := response.Capabilities[BRFCPike].(map[string]interface{}); ok {
+		var (
+			invite, outputs string
+			errMsgs         []string
+		)
+
+		if inviteStr, ok := pike["invite"].(string); ok {
+			invite = inviteStr
+		} else {
+			errMsgs = append(errMsgs, "missing invite URL in PIKE capability")
+		}
+
+		if outputsStr, ok := pike["outputs"].(string); ok {
+			outputs = outputsStr
+		} else {
+			errMsgs = append(errMsgs, "missing outputs URL in PIKE capability")
+		}
+
+		if len(errMsgs) > 0 {
+			return fmt.Errorf(strings.Join(errMsgs, "; "))
+		}
+
+		response.Pike = &PikeCapability{
+			Invite:  invite,
+			Outputs: outputs,
+		}
```

Comment:
It assumes, that the `BRFCPike` capability has to have all "sub-capabilities" enabled. 
I'm wondering, if should it be a possibility that some server -for example - supports only `"invite"` part. 🤔

Reply:
what need to be done for this/ per this comment ?

Reply:
IMO, we outputs should be required, but invite is optional.
therefore if invite is not available we shouldn't return an error.

Reply:
so that mean above code is correct, am i correct ?

Reply:
```go
if inviteStr, ok := pike["invite"].(string); ok {
			invite = inviteStr
		} else {
			errMsgs = append(errMsgs, "missing invite URL in PIKE capability")
		}
```
will change to 

```go
if inviteStr, ok := pike["invite"].(string); ok {
			invite = inviteStr
		}
```
this way only error is returned and appended per outputs

Reply:
reomoved the else block for invite